docs/design/coreclr/botr/porting-ryujit.md

```diff
@@ -86,27 +150,30 @@ emitter::emitInsTernary(instruction ins, emitAttr attr, GenTree* dst, GenTree* s
     * Sets register requirements
       * sometimes changes the register requirements children (which have already been traversed)
     * Sets the block order and node locations for LSRA
-      * LinearScan:: startBlockSequence() and LinearScan::moveToNextBlock()
+      * `LinearScan::startBlockSequence()` and `LinearScan::moveToNextBlock()`
+
+# Register Allocation
 
-## Register Allocation
 * Register allocation is largely target-independent
   * The second phase of Lowering does nearly all the target-dependent work
 * Register candidates are determined in the front-end
   * Local variables or temps, or fields of local variables or temps
   * Not address-taken, plus a few other restrictions
-  * Sorted by lvaSortByRefCount(), and marked "lvTracked"
+  * Sorted by `lvaSortByRefCount()`, and marked `lvTracked`
```

Comment:
I think this needs clarification. Not all `lvTracked` lclVars are register candidates. `lvTracked` means also that it will participate in optimizations. There are further conditions for it to be considered a register candidate.